lua/virt-column/utils.lua

```diff
@@ -13,11 +13,7 @@ end
 ---@vararg T
 ---@return T
 M.tbl_join = function(...)
-    local result = {}
-    for i, v in ipairs(vim.tbl_flatten { ... }) do
-        result[i] = v
-    end
-    return result
+    return vim.iter({ ... }):flatten():totable()
```

Comment:
Thanks, but yes let's keep this backwards compatible please.

This should be fine.

```suggestion
    ---@diagnostic disable-next-line: deprecated
    return vim.iter and vim.iter({ ... }):flatten():totable() or vim.tbl_flatten { ... }
```